src/Table/index.jsx

```diff
@@ -152,6 +152,7 @@ Table.propTypes = {
     ]).isRequired,
     columnSortable: isRequiredIf(PropTypes.bool, props => props.tableSortable),
     onSort: isRequiredIf(PropTypes.func, props => props.columnSortable),
+    hideHeader: PropTypes.bool,
```

Comment:
should this have an `isRequiredIf` as well?

Reply:
hmmmm no, no it shouldn't

Reply:
What would the condition be for requiring hideHeader?

Reply:
I guess in the same way we would require for `columnSortable`?

Reply:
Ah, okay. I thought that the ability to hide the column would be independent of whether the column could be sorted, because a sortable table could have hidden columns and a non-sortable table could have hidden columns. Maybe I'm misunderstanding?

Reply:
I could totally be misunderstanding PropTypes but wouldn't adding `isRequiredIf(PropTypes.bool, props => props.hideHeader)` be saying

> In the required `shape` there is a prop named `hideHeader`. Require this prop if it is part of our collection of props?